### PR TITLE
update gaming with facebooks

### PR DESCRIPTION
**Pull request by stephengroat**

### CODE REVIEW

**Review by Jawshy — APPROVED** [review]
`looks-good`

File: _data/gaming.yml [hunk 1]
```diff
@@ -134,7 +134,7 @@ websites:
       doc: http://itch.io/developers/two-factor-authentication
 
     - name: League of Legends
-      url: http://leagueoflegends.com
+      url: http://oce.leagueoflegends.com/
```
Comment (jamcat22) [inline _data/gaming.yml]: The old URL automatically redirects users to the correct server based on their location, for example, http://na,leagueoflegends.com for users in North America. This is by design, and no change needs to be made to the previous URL.

File: _data/gaming.yml [hunk 2]
```diff
@@ -248,7 +248,7 @@ websites:
       doc: http://forums.ubi.com/showthread.php/1463703-Secure-your-account-with-2-Step-Verification
 
     - name: Wargaming
-      url: http://wargaming.net/
+      url: http://asia.wargaming.net/
```
Comment (jamcat22) [inline _data/gaming.yml]: The old URL automatically redirects users to the correct server based on their location, for example, http://na,wargaming.net for users in North America. This is by design, and no change needs to be made to the previous URL.

**Review by jamcat22 — DISMISSED** [review]
Changes for the URLs should be reverted, because the old, non-region-specific URLs automatically redirect to the correct server based on the user's location.

**Review by Jawshy — APPROVED** [review]
Reverted 1686957212cd2eea7895458b0f2fdc0f45ac5a2b

**Comment by Jawshy** [comment]
Could another @2factorauth/collaborators please take a look at this when possible?

**Review by mxxcon — APPROVED** [review]

**Comment by stephengroat** [comment]
merging (even though it's mine) since there's 2 reviews on it

**Pull request merged**